package.json

```diff
@@ -48,7 +48,8 @@
   },
   "dependencies": {
     "algoliasearch-helper": "^3.1.0",
-    "instantsearch.js": "^4.2.0"
+    "instantsearch.js": "^4.4.0",
```

Comment:
this needs to be updated once https://github.com/algolia/instantsearch.js/pull/4359 is merged & released